<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frontcamera_dlg</class>
 <widget class="QWidget" name="frontcamera_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Front Camera</string>
  </property>
  <widget class="QGraphicsView" name="orgimage_view">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>554</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Original Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Image</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>700</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>image input mode</string>
   </property>
   <widget class="QRadioButton" name="rd_input_mode_img">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_input_mode_net">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>31</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Network</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_input_mode_loc">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>31</y>
      <width>66</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_input_mode_usb">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>31</y>
      <width>58</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>USB</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_input_mode_lcm_log_play">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LCM Log Play</string>
    </property>
   </widget>
   <zorder>rd_input_mode_net</zorder>
   <zorder>rd_input_mode_loc</zorder>
   <zorder>rd_input_mode_usb</zorder>
   <zorder>rd_input_mode_img</zorder>
   <zorder>rd_input_mode_lcm_log_play</zorder>
  </widget>
  <widget class="QPushButton" name="bt_start_stop">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>800</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Button</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>700</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Save Mode</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>175</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="ck_savemode_org">
       <property name="text">
        <string>Original</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_savemode_out">
       <property name="text">
        <string>Output</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="outimage_view">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>554</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_file_dialog">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>800</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>File Dialog</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>780</y>
     <width>1151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_image_source">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>20</width>
     <height>721</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="traffic_signal">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Traffic Signal</string>
   </property>
   <widget class="QGraphicsView" name="signal_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="signal_color_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>226</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Traffic Signal ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_traffic_img_log">
       <property name="text">
        <string>Image Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>121</y>
      <width>195</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Signal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ed_signal">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>230</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Traffic Sign</string>
   </property>
   <widget class="QGraphicsView" name="sign_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>214</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Traffic Sign ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_sign_img_log">
       <property name="text">
        <string>Image Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>261</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Pedestrian</string>
   </property>
   <widget class="QGraphicsView" name="pedestrian_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Pedestrain ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_pedestrian_img_log">
       <property name="text">
        <string>Image Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>690</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Mission</string>
   </property>
   <widget class="QCheckBox" name="ck_mission_signal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Traffic Signal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ck_mission_sign">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Traffic Sign</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ck_mission_pedestrian">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pedestrian</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ck_mission_dummy_car">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Dummy Car</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ck_mission_normal_car">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal car</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5"/>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>420</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Car</string>
   </property>
   <widget class="QGraphicsView" name="dummy_car_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="normal_car_view">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>188</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Dummy Car</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_dummy_car_img_log">
       <property name="text">
        <string>Image Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>186</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Normal Car</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_normal_car_img_log">
       <property name="text">
        <string>Image Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>21</width>
     <height>361</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>215</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_lcm_data_send">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>800</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LCM Data Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
